Dockerfiles/cluster-agent/README.md

```diff
@@ -27,9 +27,26 @@ Therefore, you will need to have docker installed on your system.
 Start by creating the binary by running `inv -e cluster-agent.build`. This will add a binary in `./bin/datadog-cluster-agent/`
 Then from the current folder, run `inv -e cluster-agent.image-build`.
 
-
 ## Running the DCA with Kubernetes
 
+### Security premise
+<a name="security-premise"></a>
+
+You can create the secret that will be used for your agents to communicate with the DCA. 
```

Comment:
Instead of `you can` (that is really vague) you should use the [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt) words in order to give a better vision upon your system, a.k.a:

1. **MUST**:   This word, or the terms "REQUIRED" or "SHALL", mean that the
   definition is an absolute requirement of the specification.

2. **MUST NOT**:  This phrase, or the phrase "SHALL NOT", mean that the
   definition is an absolute prohibition of the specification.

3. **SHOULD**:   This word, or the adjective "RECOMMENDED", mean that there
   may exist valid reasons in particular circumstances to ignore a
   particular item, but the full implications must be understood and
   carefully weighed before choosing a different course.

4. **SHOULD NOT**   This phrase, or the phrase "NOT RECOMMENDED" mean that
   there may exist valid reasons in particular circumstances when the
   particular behavior is acceptable or even useful, but the full
   implications should be understood and the case carefully weighed
   before implementing any behavior described with this label.